contrib/tetragon-rthooks/cmd/oci-hook/main.go

```diff
@@ -368,3 +375,24 @@ func getBinaryDir() string {
 
 	return path.Dir(p)
 }
+
+func serverVersion(log *slog.Logger) (string, error) {
+	req := &tetragon.GetVersionRequest{}
+	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	defer cancel()
+
+	connCtx, connCancel := context.WithTimeout(ctx, cliConf.GrpcTimeout)
+	defer connCancel()
+	conn, err := grpc.DialContext(connCtx, cliConf.AgentAddr, grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithBlock())
```

Comment:
Fyi: DialContext is deprecated and we need to move the codebase to the new way, since this is new code maybe you want to change it directly, reach out for more info!

Reply:
OK, good to know. I added a commit to replace grpc.DialContext with grpc.NewClient.